app/Jobs/TileSingleImage.php

```diff
@@ -43,14 +57,17 @@ class TileSingleImage extends Job implements ShouldQueue
     /**
      * Create a new job instance.
      *
-     * @param Image $image The image to generate tiles for.
+     * @param Image $file The image to generate tiles for.
      *
      * @return void
      */
-    public function __construct(Image $image)
+    public function __construct(Image $file)
     {
-        $this->image = $image;
-        $this->tempPath = config('image.tiles.tmp_dir')."/{$image->uuid}";
+        $this->file = $file;
+        $this->tempPath = config('image.tiles.tmp_dir')."/{$file->uuid}";
+        // for uploadToStorage method
+        $this->storage = 'image.tiles.disk';
```

Comment:
I would add this as a constructor argument. Also it's better to store the actual path, not the config key.

Reply:
resolved 4f42cb7bf35ace46d1cbeacf4f6b98fc776ec3f0